doc/zh-cn/README.md

```diff
@@ -69,3 +69,17 @@
 * [authz-keycloak](plugins/authz-keycloak-cn.md): 支持 Keycloak 身份认证服务器
 * [uri-blocker](plugins/uri-blocker.md): 根据 URI 拦截用户请求。
 * [oauth](plugins/oauth.md): 提供 OAuth 2 身份验证和自省。
+
+部署
+=======
+
+### AWS
+
+推荐的方法是在 [AWS Fargate](https://aws.amazon.com/fargate/) 上使用  [AWS CDK](https://aws.amazon.com/cdk/) 部署 APISIX，这有助于将 APISIX 层和上游层分离到具有自动缩放功能的完全托管和安全的无服务器容器计算环境之上。
```

Comment:
Is this document available for Apache APISIX 1.3? This doc is about 8 months ago.